mrr/src/main/java/gov/cdc/dex/mrr/TimerTriggerFunction.kt

```diff
@@ -20,49 +21,51 @@ import java.time.LocalDateTime
      */
     @FunctionName("PhinVocabRead")
     fun runVocab(
-        @TimerTrigger(name = "timerInfo", schedule = "0 */10 * * * *") timerInfo: String?,
+        @TimerTrigger(name = "timerInfo", schedule = "0 0 10 * * *") timerInfo: String?,
         context: ExecutionContext
     ) {
         context.logger.info("PhinVocabRead time trigger processed a request.")
-        var jedis: Jedis? = null
```

Comment:
Once the RedisUtility changes to always return Jedis, you can change the code here to this:
 RedisUtility().redisConnection().use { jedis ->
            try {
...
} finally { jedis.close() }

Reply:
done.